src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/TilbakekrevingMapper.kt

```diff
@@ -19,15 +19,15 @@ class TilbakekrevingMapper {
                     eksternFagsakId = iverksett.fagsak.eksternId.toString(),
                     personIdent = iverksett.søker.personIdent,
                     eksternId = iverksett.behandling.eksternId.toString(),
-                    behandlingstype = Behandlingstype.REVURDERING_TILBAKEKREVING,
-                    manueltOpprettet = false,
-                    språkkode = Språkkode.NB,
-                    enhetId = enhet.enhetId,
-                    enhetsnavn = enhet.enhetNavn,
+                    behandlingstype = Behandlingstype.TILBAKEKREVING, // samme som BAKS gjør
+                    manueltOpprettet = false, // manuelt opprettet ennå ikke støttet i familie-tilbake?
+                    språkkode = Språkkode.NB, // Bør følge med iverksett.søker
+                    enhetId = enhet.enhetId, // iverksett.søker.tilhørendeEnhet?
+                    enhetsnavn = enhet.enhetNavn, // iverksett.søker.tilhørendeEnhet?
```

Comment:
Lekte med tanken på å sløyfe enhetsnavn, men ut til at mye brevrelatert i familie-tilbake må sende med enhetsnavn i tillegg til nummer. Siden vi allerede må hente navkontor så kan vi sende det med for å spare familie-tilbake for et kall.